### ISSUE TITLE
Calliope support for serial write

### ISSUE THREAD

**Issue by bjost2s** [OpenRoberta/robertalab#1007]
**Is your feature request related to a problem? Please describe.**
 - As a calliope programmer you run often into problems when dealing with sensor values. Because there is no way to debug calliope so far, a serial write function would make it much easier to debug the program. 

**Describe the solution you'd like**
 - here is a good description of the feature https://mini.pxt.io/device/serial

**Comment by VinArt** [comment]
Microbit allows the usage of serial as well, if one would connect to the serial device created by microbit, the python console would be shown.

**Comment by boonto** [comment]
I started to implement the serial write for Calliope.
I moved the Arduino specific write serial up to the common robot and added the block to the Calliope expert *display* toolbox.
It works for now, however I am not sure what mode is best (`ASYNC`, `SYNC_SPINWAIT` or `SYNC_SLEEP`; see [MicroBitSerial.cpp](https://github.com/OpenRoberta/microbit-dal/blob/master/source/drivers/MicroBitSerial.cpp#L424)) and there is still some weird behaviour in combination with `wait` blocks.

Still missing:
- [ ] Write serial implementation in Microbit visitor
- [ ] Write serial implementation in simulation visitor
- [ ] Adding read serial block to mbed toolbox
- [ ] Read serial implementation in Calliope visitor 
- [ ] Read serial implementation in Microbit visitor 
- [ ] Read serial implementation in simulation visitor 
- [ ] Tests
